### DISCUSSION TITLE
Thanks

### DISCUSSION THREAD

**Discussion by sdegrande** [jothepro/doxygen-awesome-css#73] [General]
Just wanted to thank you for that really awesome doxygen theme !

**Comment by jothepro** [comment]
Awesome! I appreciate that! 😊

**Comment by Becheler** [comment]
This is indeed an amazing project!

**Comment by bavay** [comment]
Perfect, I also wanted to say a big "Thank you!!" for your work. The documentation for my project (open source) that I started 17 years ago now looks better and more modern. Thanks to you!
